app/models/draft.rb

```diff
@@ -48,7 +48,7 @@ class Draft < ApplicationRecord
   validates_uniqueness_of :dmp_id, allow_blank: true
 
   # Validate that the attachment is a PDF and that it is less than 250KB
-  validates :narrative, size: { less_than: 250.kilobytes , message: 'PDF too large, must be less than 350KB' },
+  validates :narrative, size: { less_than: 2.megabytes , message: 'PDF too large, must be less than 350KB' },
```

Comment:
Adjust the message here to `2MB` instead of `350KB`

Reply:
Oops. Thanks so much for catching that. I was going too fast and missed that one. 

I made the update.